README.md

```diff
@@ -24,5 +24,21 @@ The packaging management in this project is done using [Rye](https://rye.astral.
 1. Use `rye test --all` to run all workspaces' tests
 2. Use `rye test --package [package_name]` to run tests of specific workspace
 
+### Define a new ugbio workspace
```

Comment:
```suggestion
### Define a new ugbio workspace member
```

To be aligned with the syntax of rye/uv (and also update in the rest of the text below)